README.md

```diff
@@ -1,38 +1,315 @@
+If you wish to use just one of the following services, do not include the other in the config.
 
-**TODO: Check API diffs between these articles:** 
+```elixir
+config :sparrow, config: [fcm_config, apns_config]
+```
+**Config opts**
 
-[link](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification)
-and
-[link](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/PayloadKeyReference.html#//apple_ref/doc/uid/TP40008194-CH17-SW1)
+***FCM specific***
+- `:path_to_json` - is a path to a json file provided by FCM
 
-## Installation
+***APNS specific***
+- `:auth_type` - defines the authentication type, the allowed values are: `:token_based`, `:certificate_based`
 
-[![Build Status](https://travis-ci.com/esl/sparrow.svg?branch=master)](https://travis-ci.com/esl/sparrow)
-[![Coverage Status](https://coveralls.io/repos/github/esl/sparrow/badge.svg)](https://coveralls.io/github/esl/sparrow)
+    1 `:token_based` requres to set:
+    - `:token_id` - is a unique atom referring to a token with the same `:token_id` in config
+    
+    2 `:certificate_based` requres to set:
+    
+    - `:cert` - path to the certificate file provided by APNS
+    - `:key` - path to the key file provided by APNS
+
+***H2Worker config***
+- `:endpoint` - service uri address 
+- `:port` - service port
+- `:tls_opts` - passed to erlang [ssl](http://erlang.org/doc/man/ssl.html) module (see DATA TYPES -> ssl_option())
+- `:ping_interval` - number of miliseconds between each [ping](https://http2.github.io/http2-spec/#PING)
+- `:reconnect_attempts` - number of attempts to connect before failing the connection
+
+***Pool config***
+- `:tags` - see the [tags](#Tags) section
+- `:worker_num` - number of workers in a pool
+- `:raw_opts` - opts passed directly to [wpool](github.com/inaka/worker_pool)
+
+**FCM config**
+```elixir
+fcm_config = [fcm:
+    [
+        [
+            # Authentication
+            path_to_json: "path/to/fcm_token.json", # mandatory, path to FCM authentication JSON file
+            # H2Worker config
+            endpoint: "fcm.googleapis.com", # optional
+            port: 443, # optional
+            tls_opts: [], # optional
+            ping_interval: 5000, # optional
+            reconnect_attempts: 3, # optional
+            # Pool config 
+            tags: [], # optional
+            worker_num: 3, # optional
+            raw_opts: [] # optional, options passed directly to wpool
+        ]
+    ]
+]
+```
+**APNS config**
+```elixir
+apns_config = [apns:
+[
+    dev: [apns_pool_1, apns_pool_2 ], # list of apns_pool_configs by default set to APNS development endpoint, is a list of APNS pools
+    prod: [apns_pool_3 ],  # list of apns_pool_configs by default set to APNS production endpoint, is a list of APNS pools
+    tokens: [apns_token_1, apns_token_2 ] # optional, is a list of APNS tokenss
+]
+```
+***APNS pool***
```

Comment:
Alright, so APNS has "APNS config" and "APNS pool", but FCM only has "FCM config"... doesn't FCM have pools?

Reply:
No, FCM does not yet support more than one pool. This feature is not included in first version, but config is is written in a way that will be compatible when it will be implemented.

Reply:
Looks like information that should be included in the readme!